packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md

```diff
@@ -5,7 +5,6 @@ category: management
 
 # MyAccount API
 
-<ApiLifecycle access="ea" /></br>
 <ApiLifecycle access="ie" />
```

Comment:
Not familiar with this at all, just curious, should "ie" be "ga"?

Reply:
No, that's the Identity Engine flag.

<img width="715" alt="Screen Shot 2022-07-06 at 4 14 19 PM" src="https://user-images.githubusercontent.com/45509385/177652485-2ad20a4e-5273-44b9-98d2-f3c7f2d86240.png">